docs/ossm/README.md

```diff
@@ -6,4 +6,80 @@ This documentation is specific to the OpenShift Service Mesh product and may dif
 
 # Table of Contents
 
-TODO
+- [Running OSSM 2 and OSSM 3 side by side](#running-ossm-2-and-ossm-3-side-by-side)
+
+
+# Running OSSM 2 and OSSM 3 side by side
```

Comment:
Just an FYI that "OSSM" isn't a public acronym we can use in product docs. It's not a big deal here, as the doc team uses a parameter for the product name so that it's easy for them to do search/replaces (if it ever changes), but perhaps expand it the first time you use it "Red Hat OpenShift Service Mesh (OSSM)".

Reply:
Done